### PR TITLE
docs(website): add 'Getting Started' section

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

This PR resurrects the old getting started section and changes is to fit the current publishing workflow.

## Test Plan

Deployed version on cloudflare

### CODE REVIEW

File: website/src/_includes/docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Getting Started
+
+Install Rome via one of the following alternatives.
+
+### Installation of the CLI
+
+You can install the `rome` binary via any node package manager.
+
+> WARNING: we strongly suggest to **not** install the binary globally, instead install the binary
```
Comment (leops) [inline website/src/_includes/docs/getting-started.md:9]: Maybe we could direct people that just want to try out the CLI to use `npx rome@next` instead of having to add it to their `package.json` upfront

File: website/src/_includes/docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+> locally to your project and create local commands in order to leverage the CLI.
+
+#### [Installation via Yarn](https://yarnpkg.com/)
```
Comment (leops) [inline website/src/_includes/docs/getting-started.md:12]: It would be nice if the website / theme had a "tabbed widget" to collapse these kind of "platform specific" examples into a single section

Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:12]: We could look into it later. We don't have such feature for now (it would require some HTML/CSS/JS work). At least, like this, each section has its own link.

File: website/src/_includes/docs/getting-started.md (hunk 1 above)
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:9]: Does `yarn` have something like `npx`?

Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:9]: `yarn` v2 has `yarn dlx`, it seems. I will add only `npx/pnpx` for now. I know the community is quite divided on yarn v1 and v2.

File: website/src/_includes/docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+## Getting Started
+
+Install Rome via one of the following alternatives.
+
+### Installation of the CLI
```
Comment (MichaReiser) [inline website/src/_includes/docs/getting-started.md:5]: I would rephrase the title to be more scenario centered: 

```
Use Rome's CLI
```

File: website/src/_includes/docs/getting-started.md (hunk 2 above)
Comment (MichaReiser) [reply website/src/_includes/docs/getting-started.md:12]: It feels a bit excessive to have a new section for each manager. How about a table? 

```md
| Package Manager | Instructions | 
| npm | npm i rome@next |
| pnpm | pnpm i rome@next |
| yarn| yarn add rome@next |
```

File: website/src/_includes/docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+```bash
+pnpm i rome@next
+```
+
+### Installation VS Code extension
+
+You can use Rome by downloading the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=rome.rome) from the marketplace. 
+
+
+### Installation via CI
```
Comment (MichaReiser) [inline website/src/_includes/docs/getting-started.md:35]: Nit:
```suggestion
### Use Rome in your CI pipeline
```

File: website/src/_includes/docs/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+You can install Rome in a continues integration too.
```
Comment (MichaReiser) [inline website/src/_includes/docs/getting-started.md:37]: I would leave out this sentence. It doesn't add more information than the sections following.

File: website/src/_includes/docs/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+#### Installation on GitHub
```
Comment (MichaReiser) [inline website/src/_includes/docs/getting-started.md:39]:
```suggestion
#### GitHub Actions
```

File: website/src/_includes/docs/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+Please refer to the [official page](https://github.com/rome/setup-rome#usage) for more information about its configuration:
```
Comment (MichaReiser) [inline website/src/_includes/docs/getting-started.md:41]:
```suggestion
Please refer to the documentation of the [`setup-rome` action](https://github.com/rome/setup-rome#usage) for more information about its configuration: 
```

File: website/src/_includes/docs/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+        version: latest 
+    - run: rome --help
+```
+
+#### Installation on any other CI
+
+You can download and install the binary directly using `curl`:
+
+```shell
+curl -L https://github.com/rome/tools/releases/download/latest/rome-linux-x64 -o rome
```
Comment (MichaReiser) [inline website/src/_includes/docs/getting-started.md:56]:
```suggestion
curl -L https://github.com/rome/tools/releases/download/latest/rome-<OS>-<ARCH> -o rome
```

File: website/src/_includes/docs/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+chmod +x rome
+```
+
+Please make sure to choose the correct architecture from the [releases page](https://github.com/rome/tools/releases).
+
+
+## After installation
+
+- check the [formatter section](/formatter#use-the-formatter-with-the-cli) for options and commands;
+- check the options available in the [VS Code extension](/formatter#use-the-formatter-with-the-vscode-extension)
```
Comment (MichaReiser) [inline website/src/_includes/docs/getting-started.md:66]: I would move this to the editor section. It isn't meaningful for someone that just installed the CI.

File: website/src/_includes/docs/getting-started.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+npm i rome@next
+```
+
+#### [Installation via pnpm](https://pnpm.io/)
+
+```bash
+pnpm i rome@next
+```
+
+### Installation VS Code extension
```
[the remark this answers is not in the document]
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:30]: > Do we need to mention anything in case prettier is installed too and how they can make Rome the preferred formatter?

Good point! I think we explain that in the `formatter.md`, where we have the `### Use the formatter with the VSCode extension` section.

File: website/src/_includes/docs/getting-started.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
+## Getting Started
+
+Install Rome via one of the following alternatives.
```
[the remark this answers is not in the document]
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:3]: I can't find a section where, for example, `deno` mention that. (as they use Rust and they are bound to it for architecture support)

We could add a small paragraph saying that we don't support all the architectures and to check the release page? Not sure how phrase it.

File: website/src/_includes/docs/getting-started.md (hunk 3 above)
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:5]: I wanted to keep a link that "talks", descriptive. What about `Install Rome's CLI`?

File: website/src/_includes/docs/getting-started.md (hunk 9 above)
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:66]: The things is, once we installed Rome (regardless what), we need direct new users to somewhere. We can't assume that they know what to do/where to go after the installation. How would you guide new users?

File: website/src/_includes/docs/getting-started.md (hunk 3 above)
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:5]: And it's not actually "use", we are actually installing it... 🤔

File: website/src/_includes/docs/getting-started.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (MichaReiser) [reply website/src/_includes/docs/getting-started.md]: Excellent point. We could inspire our own getting started on how deno structures their guide. 

That's what I found at the top of [their guide](https://deno.land/manual/getting_started/installation.md0 and we could use a very similar copy saying that we support macOS, Linux, and Windows, both x64 and ARM64.

> Deno works on macOS, Linux, and Windows. Deno is a single binary executable. It has no external dependencies.

> On macOS, both M1 (arm64) and Intel (x64) executables are provided. On Linux and Windows, only x64 is supported.

File: website/src/_includes/docs/getting-started.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+#### Installation on any other CI
+
+You can download and install the binary directly using `curl`:
+
+```shell
+curl -L https://github.com/rome/tools/releases/download/latest/rome-<OS>-<ARCH> -o rome
+chmod +x rome
+```
+
+Please make sure to choose the correct architecture from the [releases page](https://github.com/rome/tools/releases).
```
Comment (leops) [inline website/src/_includes/docs/getting-started.md:49]: We could expand on that by adding the "supported platforms" table here, explaining that `<OS>` and `<ARCH>` follow the node.js / VScode syntax, and the accepted values are `win32`, `darwin` and `linux` for `<OS>`, and `x64` and `arm64` for `<ARCH>` (this is different from the "target triple" format used by LLVM and Rust for example where `x64` is called `x86_64` and `win32` is called `windows`)

Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:67]: Please check the new section and if it's how you imagined. I would have added a link to every single distribution but their link has the version number. If we had something like `https://github.com/rome/tools/releases/download/latest/rome_lsp-darwin-arm64.vsix` (notice the `/latest/` instead of the version number), we could point to them to a link.

File: website/src/_includes/docs/getting-started.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md]: I updated the section, let me know if it's fine

File: website/src/_includes/docs/getting-started.md (hunk 12 above)
Comment (MichaReiser) [reply website/src/_includes/docs/getting-started.md:67]: You can use latest to get the latest release: `https://github.com/rome/tools/releases/latest/download/rome-linux-x64`

**Review by MichaReiser — APPROVED** [review]

File: website/src/_includes/docs/getting-started.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
+## Getting Started
+
+Rome works on Linux, macOS, and Windows operative systems. 
+
+On macOS, both M1 (arm64) and Intel (x64) executables are provided. On Linux and Windows, only x64 is supported.
```
Comment (leops) [inline website/src/_includes/docs/getting-started.md:5]: Actually we do support ARM CPUs on Windows and Linux too

File: website/src/_includes/docs/getting-started.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+Where `OS` and `ARCH` follow the Node.js syntax convention:
+
+- `OS` is one of the following values: `win32`, `darwin` or `linux`
+- `ARCH` is one of the following values: `arm64` or `x64`
+
+
+|         | `win32`       | `darwin`       | `linux`       |
+|---------|---------------|----------------|---------------|
+| `arm64` | `win32-arm64` | `darwin-arm64` | `linux-arm64` | 
+| `x64`   | `win32-x64`   | `darwin-arm64` | `linux-arm64` |
```
Comment (leops) [inline website/src/_includes/docs/getting-started.md:65]:
```suggestion
| `x64`   | `win32-x64`   | `darwin-x64`   | `linux-x64`   | 
```

File: website/src/_includes/docs/formatter.md [hunk 15]
```diff
@@ -28,6 +28,10 @@ This is an **opt-in feature** that allows the developers to experiment how a for
 
 > WARNING: all options are marked as **BETA** because this might change, once we will add support of a configuration file
 
+If you want to set Rome formatter as your default formatter, you can do so by opening the [command palette]
+and select `Forma With ...` , then `Configure Default Formatter` and finally select `Rome`.
```
Comment (leops) [inline website/src/_includes/docs/formatter.md:32]: We should probably use the exact wording of the command (`Format Document With ...`), and mention the user has to be in a JS / TS file for the command to appear

Comment (ematipico) [reply website/src/_includes/docs/formatter.md:32]: I will phrase like this: "The option will appear only for documents that Rome supports."

So when we will have JSON formatting, this phrase already covers it.

File: website/src/_includes/docs/getting-started.md (hunk 12 above)
Comment (ematipico) [reply website/src/_includes/docs/getting-started.md:64]: > You can use latest to get the latest release: `https://github.com/rome/tools/releases/latest/download/rome-linux-x64`

Shoot! I saw this comment just now! I will fix it with another PR
